src/ctia/http/routes/crud.clj

```diff
@@ -229,45 +231,59 @@
                 :auth-identity identity
                 :identity-map identity-map
                 (GET "/histogram" []
-                     :return [{:key s/Str :value s/Int}]
+                     :return AggResult
                      :summary (format "Histogram for a %s field" capitalized)
                      :query [params histogram-q-params]
```

Comment:
why is this a lowercase symbol here and not a Camelcase named symbol as we use for all schemas?

Reply:
`histogram-q-params` is built at the beginning of the function like stated in my previous comments.
It's similar to `[search-q-params](https://github.com/threatgrid/ctia/blob/master/src/ctia/http/routes/crud.clj#L213)` that is passed to the function.

Reply:
Doesn't it mess with swagger-UI?

Reply:
I just meant, I think a `{SearchQueryParameters :search-q-params}` would do the trick for the macro.

Reply:
That's a query parameters here, it's not the same behavior than for the `body` parameter of a `POST`.
In swagger it produces:
![Screenshot 2020-05-04 at 15 34 01](https://user-images.githubusercontent.com/3592199/80971256-c4311180-8e1c-11ea-8311-ef09860ba0b2.png)
In the case of a body, it uses the Schema name but it's not dependant of the variable name:
`:body [new-entity new-schema {:description (format "a new %s" capitalized)}]`
![Screenshot 2020-05-04 at 15 32 40](https://user-images.githubusercontent.com/3592199/80971402-02c6cc00-8e1d-11ea-98ac-c06990c8cdfb.png)